python/src/otel/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG runtime=python3.11
+ARG runtime=python3.8
```

Comment:
Why do we set the runtime back to python3.8?

Reply:
My first lambda test functions were Python 3.8 so I made this working change to accommodate and use the correct extension .so inside the layer-python used in the lambda environment.

Moving forward we'll need to support Python 3.8 to 3.11. Otel does proper builds using [template](https://github.com/solarwinds/opentelemetry-lambda/blob/NH-59205-apm-python-custom-layer/python/src/template.yml) and Makefile on GH I think? Not with this local setup.